walkthroughs/eks-getting-started/infrastructure/appmesh_templates/appmesh-yelb-ui.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       app: yelb-ui
   listeners:
     - portMapping:
-        port: 4567
+        port: 80
```

Comment:
How does this affect TLS?

Reply:
Container was exposing port 80 but the virtual node was being accessed under the port 4567. Had to fix this.

Reply:
Neither of changes add directly support for TLS. However I'm working on App Mesh TLS blog post which needs to have proper base setup (without TLS) of the sampe yelb app already integrated with App Mesh. The changes here are to fix base configuration and prepare it for further integration with TLS which will be separate code.

1. change of yelb-ui version from 0.6 to 0.7 adds https suppor for this component. Refer here: https://github.com/mreferre/yelb/commit/048d6db7218fac7023137de018551539baa8ec1b

2. change of yelb-ui virtual node listener port from 4567 to 80 is just fix which set correct port to expose yelb-ui

Both changes were tested by deployment of the yelb app with new settings.